sympy/physics/control/lti.py

```diff
@@ -1134,6 +1138,8 @@ def __rsub__(self, other):
         return -self + other
 
     def __mul__(self, other):
+        if isinstance(other, Feedback):
+            other = other.doit()
```

Comment:
Here `doit()` is needed because otherwise it gives an error while multiplying a `Feedback` object to a `TransferFunction` object.

Reply:
Am I not supposed to not use `doit()` here also ?

Reply:
I don't think` doit() `should be used internally at any place.

Reply:
`doit()` for `Feedback` should not be used but for other objects it is already used eg:
https://github.com/sympy/sympy/blob/101e6d0215b5ada022577f1b77e512e54f24b59e/sympy/physics/control/lti.py#L2465